File: app/src/main/java/com/taras/movieapp/MovieApplication.java
package com.taras.movieapp;

import android.app.Application;

import com.facebook.stetho.Stetho;

public class MovieApplication extends Application {

    public static MovieApplication sInstance;

    public MovieApplication() {
        sInstance = this;
    }

    public static MovieApplication getMovieAppContext() {
        return sInstance;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        Stetho.initializeWithDefaults(this);
    }
}

File: app/src/main/java/com/taras/movieapp/mvvm/MovieApplication.kt
package com.taras.movieapp.mvvm

import android.app.Application
import com.facebook.stetho.Stetho

class MovieApplication : Application() {

    companion object {
        lateinit var sInstance: MovieApplication
            private set

        val getAppInstance: MovieApplication
            get() = sInstance
    }

    override fun onCreate() {
        super.onCreate()
        sInstance = this
        Stetho.initializeWithDefaults(this)
    }
}